src/main/kotlin/cacophony/controlflow/generation/CFGGenerator.kt

```diff
@@ -27,10 +27,13 @@ internal class CFGGenerator(
     private val sideEffectAnalyzer = SideEffectAnalyzer(analyzedUseTypes)
     private val operatorHandler = OperatorHandler(cfg, this, sideEffectAnalyzer)
     private val assignmentHandler = AssignmentHandler(cfg, this)
+    private val functionHandler = functionHandlers[function]!!
+    private val prologue = listOfNodesToExtracted(getCurrentFunctionHandler().generatePrologue())
```

Comment:
Yeah, I introduced it because i had not noticed getCurrentFunctionHandler function already existed. I think I will stick to what is already there, as it also handles possibile errors more verbosely